src/switchboard_accounts.erl

```diff
@@ -67,16 +67,12 @@ init({ConnSpec, Auth, Mailboxes}) ->
     true = gproc:reg(switchboard:key_for(Account, account)),
     RestartStrategy = one_for_all,
     MaxR = MaxT = 5,
-    ActiveChildSpec = {active,
-                       {imap, start_link,
-                        [ConnSpec,
-                         [{cmds, [{cmd, {call, {login, Auth}}}]},
-                          {post_init_callback,
-                           switchboard:register_callback(Account, active)}]]},
-                       permanent,
-                       5000,
-                       worker,
-                       [imap]},
+    ActiveChildSpec = poolboy:child_spec(
+                        Account,
+                        [{name,
+                          {via, gproc, switchboard:key_for(Account, pool)}},
+                         {worker_module, imap}, {size, 4}, {max_overflow, 5}],
```

Comment:
What do you think about allowing the pool size to be configurable? 4 IMAP connections means that switchboard will be holding open a minimum of 5 per account, which is pretty heavy.